Ginger/GingerCore/Drivers/SeleniumDriver.cs

```diff
@@ -401,10 +401,13 @@ public override void StartDriver()
                         if (SeleniumUserArgs != null)
                             foreach (string arg in SeleniumUserArgs)
                                 options.AddArgument(arg);
-                        if (HideConsoleWindow)
+                        if (HideConsoleWindow || HttpServerTimeOut != 60)
```

Comment:
why we checking if servertimeout is not 60 ?

Reply:
I have pushed changes for above. Please review.